src/dropdown/dropdown.scss

```diff
@@ -213,23 +213,59 @@ $height-small: 30px !default;
   }
 
   &.Select--multi {
+    height: auto;
+
     &.has-value .Select-input {
       position: relative;
     }
 
+    &.has-value .Select-control {
+      padding: 0;
+    }
+
+    .Select-placeholder {
+      padding: 0 10px;
+    }
+
+    .Select-control {
+      padding: 0;
+      display: table;
```

Comment:
Changing the `display` value to table is really dangerous, since we're forcing the whole dropdown to use `display: flex` (see the `normalize` mixin defined above) . You should be able to accomplish your task using `@include flex_flex-wrap(wrap)` where needed